docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
-Get started
+Get Started
 ===========
 
+.. contents::
+    :depth: 2
+    :local:
+
+
+
 Install CumulusCI
 -----------------
 
+.. note:: These installation instructions assume some familiarity with entering commands into a terminal.
+    If that's completely new to you, we recommend visiting the `CumulusCI Setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ on Trailhead for a step by step walkthrough.
+
+
+
+On macOS
+^^^^^^^^
+
+`Homebrew <https://brew.sh/>`_ is a prerequisite for installing CumulusCI on macOS. Start there.
+
+
+Install via ``pipx``
+********************
+
+``pipx`` ensures that CumulusCI and its dependencies are installed into their own Python environment separate from other Python software on your computer. **We cannot recommend it enough!**
+
+First, install ``pipx`` with the following commands::
+
+    $ brew install pipx
+    $ pipx ensurepath
+
+After ``pipx`` installs, install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished `verify your installation`_.
+
+
+
+On Linux
+^^^^^^^^
+
+Install via ``pipx``
+********************
+
+``pipx`` ensures that CumulusCI and its dependencies are installed into their own Python environment separate from other Python software on your computer. **We cannot recommend it enough!**
+
+Installation instructions for ``pipx`` can be found `here <https://pipxproject.github.io/pipx/installation/>`_.
+
+After ``pipx`` installs, install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished `verify your installation`_.
+
+
+
+On Windows
+^^^^^^^^^^
+
+Install Python 3
+********************
+
+#. Go to the `Python downloads page <https://www.python.org/downloads/release/python-383/>`_.
+#. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
```

Comment:
```suggestion
#. Download the latest Python 3 release. Most users select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it depends on your particular computer setup.
```
Avoid modal verbs like "may", "should", and "might": https://confluence.internal.salesforce.com/display/DOCTEAM/modal+verbs